src/internal/runner.csx

```diff
@@ -34,10 +35,13 @@ public static class SimpleTargetsRunner
                     showList = true;
                     break;
                 case "-n":
-                    dryRun = true;
+                    options.DryRun = true;
                     break;
                 case "--no-color":
-                    color = false;
+                    options.Color = false;
+                    break;
+                case "--skip-dependencies":
+                    options.RunDependencies = false;

```

Comment:
Why not name the property `SkipDependencies` (and obviously reverse the polarity)?

Reply:
I conformed to the way Color was implemented. Happy to switch.

Reply:
Fair point. Perhaps `--no-color` should be flipped too (in another PR).

Reply:
Sure. I'll make the SkipDependencies change.